auth-web/tests/unit/views/RequestNameView.spec.ts

```diff
@@ -12,10 +12,15 @@ const vuetify = new Vuetify({})
 // Prevent the warning "[Vuetify] Unable to locate target [data-app]"
 document.body.setAttribute('data-app', 'true')
 
+const mockSession = {
+  'NRO_URL': 'Mock Url'
```

Comment:
Same comment/question as in HomeView about testing this.

Reply:
Totally, its a great idea. Will do this, in a future commit ( If thats ok :) )